modules/chapter1/pages/section3.adoc

```diff
@@ -3,17 +3,18 @@
 
 In this exercise, you will use RHOAI to train and evaluate a simple machine learning model, by using JupyterLab and other common ML libraries.
 
+// Not understood the demo. Can be moved in next chapter before Lab.
```

Comment:
OK. Good suggestion. Please cut the demo from here and move it to the appropriate place before final Lab